plugins/pay.c

```diff
@@ -1704,8 +1704,9 @@ static void init(struct plugin *p,
 }
 
 #if DEVELOPER
-struct payment_modifier *paymod_mods[2] = {
+struct payment_modifier *paymod_mods[3] = {
```

Comment:
explicit size not required here, BTW

Reply:
I'll omit them going forward, it was one of the main causes of conflict while rebasing actually :-)